nvflare/private/fed/server/job_cmds.py

```diff
@@ -407,12 +407,20 @@ def abort_job(self, conn: Connection, args: List[str]):
         try:
             job_id = conn.get_prop(self.JOB_ID)
             with engine.new_context() as fl_ctx:
-                message = job_runner.stop_run(job_id, fl_ctx)
-                if message:
-                    conn.append_error(message, meta=make_meta(MetaStatusValue.INTERNAL_ERROR, message))
-                else:
-                    conn.append_string("Abort signal has been sent to the server app.")
+                job_manager = engine.get_component(SystemComponents.JOB_MANAGER)
+                job = job_manager.get_job(job_id, fl_ctx)
```

Comment:
You shouldn't need to make this call. You could simply do:

job = conn.get_prop(self.JOB).

This is because the job authorizer already did job_manager.get_job() for you.

Reply:
changed.